tests/multigrid/test_embedded_transfer.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from firedrake import *
+
+
+solver_parameters = {
+    "mat_type": "aij",
+    "snes_type": "ksponly",
+    "ksp_type": "cg",
```

Comment:
Should this be `fcg`  for the case `exact=True`, as you're doing a solve with CG inside the transfer?

Reply:
Do you mean `use_fortin_interpolation=False`? I suspect it should be `fcg` in that case, because yes, the V-cycle is no longer a linear operator.

Reply:
Yes, sorry, I meant `use_fortin_interpolation=False` (I used to have `exact=True` in my code, called it that, then edited one side of the equation but not the other)

Reply:
Done in f26d11ac.